scripts/accounts/gcloud/setup-firebase.sh

```diff
@@ -294,6 +299,12 @@ createPlatformApps() {
     fi
 }
 
+nextSteps() {
+    echo "Next steps:"
+    echo "1- Accept consent screen: https://console.cloud.google.com/apis/credentials/consent?project=$projectName"
+    echo "2- Go to https://console.cloud.google.com/google/maps-apis/api-list?project=$projectName, choose Maps Static API, then go to Credentials and copy the current secret"
```

Comment:
```suggestion
    echo "1. Accept consent screen: https://console.cloud.google.com/apis/credentials/consent?project=$projectName"
    echo "2. Go to https://console.cloud.google.com/google/maps-apis/api-list?project=$projectName, choose Maps Static API, then go to Credentials and copy the current secret."
```

Is it clear enough to the user what does he have to do with the secret? Personally, I don't understand it.